dkg/dkg.go

```diff
@@ -608,6 +669,58 @@ func signDepositMsgs(shares []share, shareIdx int, withdrawalAddresses []string,
 	return set, msgs, nil
 }
 
+// signValidatorRegistrations returns a partially signed dataset containing signatures of the validator registrations signing root.
+func signValidatorRegistrations(shares []share, shareIdx int, feeRecipients []string, gasLimit uint64) (core.ParSignedDataSet, map[core.PubKey]core.VersionedSignedValidatorRegistration, error) {
+	const registrationTime string = "Jan 1, 2000"
```

Comment:
what about `time.Date` to avoid parsing, also maybe move it to global variable (like. a constant)